tests/projected_gradient_test.py

```diff
@@ -28,6 +28,8 @@
 import numpy as onp
 
 
+n_calls = 0
```

Comment:
same comment as in the other file: uppercase global variables and docstring for tests

Reply:
done